Assets/Scripts/Game/DaggerfallExteriorAutomap.cs

```diff
@@ -762,9 +763,34 @@ private void createBuildingNameplates(DFLocation location)
                             PlayerGPS.DiscoveredBuilding discoveredBuilding;
                             if (GameManager.Instance.PlayerGPS.GetDiscoveredBuilding(buildingSummary.buildingKey, out discoveredBuilding))
                             {
-                                // only create nameplates for buildings different than "Residence"
-                                if (discoveredBuilding.displayName != "Residence")
+                                
+                                if (!RMBLayout.IsResidence(buildingSummary.BuildingType)) // guilds, shops, taverns, palace (general case)
```

Comment:
@Nystul-the-Magician Nystul, I updated post in forums. Please feel free to no longer filter residences from automap. I know that I've been adamant about keeping map free of non-functional clutter, but tying the automap in with all the other game systems that can modify nameplates (especially since mods can do this) is going to be much too hard bordering on impossible. The automap shouldn't have to worry about this.

I think the best solution is is just for me to admit that it's best to do things like classic here and just display nameplate as set by discovery system. If that nameplate is a non-useful residence then so be. This is how classic works anyway, and I doubt many people will complain. It will also allow users to potentially edit any nameplates on automap as well.

I can admit when my decisions are wrong and just make things more difficult for you guys. This is one of those cases. :)

Reply:
what do you think about this:
one way to go would be to filter indeed by name "Residence" but get this string from recently introduced ConversationText.txt file. Translators would then be cautious to translate this correctly but it would work.
where does the building name creation / discovery mechanism get the string "Residence" from? maybe best thing would to make a common mechanism that all systems use for deriving string "Residence" and automap could then filter it. Of course this mechanism must support translations